app/Http/Controllers/Service/Admin/TradersController.php

```diff
@@ -27,13 +29,7 @@ class TradersController extends Controller
      */
     public function index()
     {
-        $traders = Trader::with(['users', 'market'])->get();
```

Comment:
Interesting, that's my sorting code :-(

Reply:
I've moved that code in a private method - `getTraders()` - inside the controller as I'm using it in the download method, too and wanted to avoid duplication. I'm referring to that code in the index as `$traders = $this->getTraders()` 🤔

Reply:
ah! cool :-)